packages/scaffold/src/views/w3m-account-settings-view/index.ts

```diff
@@ -246,9 +246,9 @@ export class W3mAccountSettingsView extends LitElement {
     await ConnectionController.reconnectExternal(emailConnector)
 
     this.text =
-      this.preferredAccountType === W3mFrameRpcConstants.ACCOUNT_TYPES.SMART_ACCOUNT
-        ? 'Switch to your smart account'
-        : 'Switch to your EOA '
+      accountTypeTarget === W3mFrameRpcConstants.ACCOUNT_TYPES.SMART_ACCOUNT
```

Comment:
Oh how we didn't notice before 🤔

Reply:
Seemed to work great locally, but in lab it didn't. This should solve it.